handbook/engineering/batch-changes/ce-onboarding.md

```diff
@@ -0,0 +1,62 @@
+# Batch Changes CE onboarding
+
+Welcome to Batch Changes! This page is primarily for new members of the CE team, but it could be helpful to other teams, too! After going through these steps, you should:
+
+- understand what Batch Changes is
+- be able to present it to customers from a value, usage, and functional perspective
+- be comfortable demoing Batch Changes to developers and answering questions
+
+## Prerequisites
+General [CE onboarding](../../ce/onboarding.md)
+
+## Part 1: Discover Batch Changes
+- Watch the [demo video](https://www.youtube.com/watch?v=eOmiyXIWTCw) and checkout the [landing page](https://about.sourcegraph.com/batch-changes/)
+- What is Batch Changes, and why does it matter to customers? Read about [product positioning and messaging](../../marketing/batch_changes_positioning.md)
+- How to demo Batch Changes? Watch the [Batch Changes deep dive](https://docs.google.com/presentation/d/1CN3KQf1Hfdb4RO6FgBgKuiHK4ERcOAHPgVnOcBu-MPU/edit#slide=id.ga366db8d9b_0_116)
+- How does Batch Changes work? Read the Batch Changes documentation section of the [Supporting Batch Changes guide](./supporting-batch-changes.md#batch-changes-documentation)
+
+
+## Part 2: Use Batch Changes
+
+- If you are blocked at any time during onboarding, ask for help in [#batch-changes](https://sourcegraph.slack.com/archives/CMMTWQQ49).
+- Feel free to document your onboarding experience and share it back with the Bacth Changes team so that we can make this even better!
+- In this tutorial, we will use [k8s.sgdev.org](https://k8s.sgdev.org) as our environment. You should have access by default using SSO.
+- Tip: Use the docs extensively to complete the tutorial steps.
+
+### Step 1: Quickstart
+Let's start with a quick setup example to warm up: [Quickstart](https://docs.sourcegraph.com/batch_changes/quickstart).
+
+Answer those questions:
+- what does `on` do?
+- what are `steps`?
+- what is the difference between *apply* and *publish*?
+
+### Step 2: comby
+Let's use [comby](https://comby.dev/). Batch Changes can run any code change tool, but comby is great for structural-type change. See how it works with Batch Changes in this [tutorial](https://docs.sourcegraph.com/batch_changes/tutorials/refactor_go_comby). Don't publish your changesets yet!
+
+Answer those question:
+- what is comby?
+- does Batch Changes rely on comby?
+
+
+### Step 3: The `on` attribute
+Can you scope down your batch change to only a few repos? Modify the spec to scope down your batch change to <repo1> <repo2> <repo3>. Tip: you may find [`on`](https://docs.sourcegraph.com/batch_changes/references/batch_spec_yaml_reference#on) useful.
```

Comment:
Do we want to tell them to do Sourcegraph repos instead of repo1, repo2, repo3 (which I'm assuming aren't literal repo names you've set up?)

Reply:
Also IMO it might be worth it to move the "limit the repos" thing to step 2, comby to step 3—if you don't scope the repos it takes foreEVER for batch changes to run, depending on what's on the instance. (When I was making that 2to3 change it kept falling over because it was trying to refactor ctags.)

Reply:
Prior to merging this, I will create (and index) 3 repositories mirrors that folks can actually open PRs on. Don't know if that makes sense. I've seen many occurrences of "I don't want to actually publish the changesets because it'll be spam on the repo"

Reply:
I think that's fine, but I think they should just be limiting to those 3 repos from the jump, if possible. Even if it's just doing the preview, it is slower to run across many repos, so they'll have an easier time if their specs are always limited to those 3 repos from the start.

Reply:
Oh. Haddn't thought out that. What I'll do is
- [ ] create 3 repo mirrors called `ce-onboarding-<something>` 
- [ ] add instructions to add `repo: ce-onboarding` to the `on.repositoriesMatchingQuery` 

That way, the oboardee (or whatever the term) discovers `on.repositoriesMatchingQuery`, then `on.repo`. 
WDYT?

Reply:
✅